components/Header.tsx

```
@@ -136,43 +207,23 @@ export default function Header({ siteTitle }: { siteTitle: string }) {
-                                <a
-                                    key={social.name}
-                                    href={social.href}
-                                    target="_blank"
-                                    rel="noopener noreferrer"
-                                    className="text-base-content/70 hover:text-primary transition-colors"
-                                    aria-label={social.name}
-                                    onClick={() => setMobileMenuOpen(false)}
-                                >
-                                    {social.icon}
-                                </a>
+                                <div key={social.name} onClick={closeMenu}>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)